testsuite/integration-arquillian/test-apps/servlets/pom.xml

```diff
@@ -14,11 +14,8 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.jboss.spec.javax.servlet</groupId>
-            <artifactId>jboss-servlet-api_3.0_spec</artifactId>
-            <!-- wherever this is being deployed, the Servlet API spec should already be there.
-            Therefore, make this provided to avoid mixing APIs on the target -->
-            <scope>provided</scope>
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>jakarta.servlet-api</artifactId>
```

Comment:
Likewise this dependency is also not versioned anywhere explicitly.

Reply:
I suppose the same here. But this one is using the quarkus bom from the parent bom.xml. If the adapters are separated to another project this would need to use direct versioning but for the moment I would use the quarkus bom as today.